theano/tensor/tests/test_basic.py

```diff
@@ -3750,7 +3752,10 @@ def get_mat(s1, s2):
             self.assertRaises(ValueError, f, get_mat(3, 4), get_mat(3, 4),
                               get_mat(2, 5))
```

Comment:
Remove the `get_mat(...)` arguments here, since `f` does not take arguments anymore.